disk_test.go

```diff
@@ -0,0 +1,113 @@
+package vz_test
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/Code-Hex/vz/v3"
+)
+
+func TestCreateSparseDiskImage_FileCreated(t *testing.T) {
+	if vz.Available(26) {
+		t.Skip("CreateSparseDiskImage is supported from macOS 26")
+	}
+
+	dir := t.TempDir()
+	path := filepath.Join(dir, "sparse_disk.img")
+
+	ctx := context.Background()
+	size := int64(1024 * 1024 * 1024) // 1 GiB
+
+	err := vz.CreateSparseDiskImage(ctx, path, size)
+	if err != nil {
+		t.Fatalf("failed to create sparse disk image: %v", err)
+	}
+
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		t.Fatal("disk image file was not created")
+	}
+}
+
+func TestCreateSparseDiskImage_ASIFFormat(t *testing.T) {
+	if vz.Available(26) {
+		t.Skip("CreateSparseDiskImage is supported from macOS 26")
+	}
+
+	dir := t.TempDir()
+	path := filepath.Join(dir, "sparse_disk.img")
+
+	ctx := context.Background()
+	size := int64(1024 * 1024 * 1024) // 1 GiB
```

Comment:
Occupying 1 GB for testing is not good - can't you do something like 1 KB, 1 MB?

Reply:
@nagypeterjob I see Thanks!